e2e/page-objects/mapPage.ts

```
@@ -60,43 +80,64 @@ export class MapCanvas extends HelperBase {
+    (args) =>
+      `Click with left click on the map view at ${args[0]} pixels for X and ${args[1]} pixels for Y`,
+  )
   async clickPlaceOnMapView(pixelsForX: number, pixelsForY: number) {
     const map = this.page.locator('#map-view');
     const box = await map.boundingBox();
-    expect(box).not.toBeNull();
+    expect(
+      box,
+      'Expect map view element to be displayed and be parseable to Playwright bounding box',
+    ).not.toBeNull();
     await this.page.mouse.click(box!.x + pixelsForX, box!.y + pixelsForY, { delay: 330 });
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)